src/software/gui/drawing/robot.h

```diff
@@ -17,10 +17,12 @@
  * @param scene The scene to draw on
  * @param position The position of the robot
  * @param velocity The velocity of the robot
- * @param color The color to draw the robot's velocity
+ * @param slow_colour The velocity line colour when the velocity is small
+ * @param fast_colour The velocity line colour when the velocity is fast
```

Comment:
Contrasting between 'small' and 'fast' velocity in the comment seems strange. Change to "when the velocity is slow" to keep it consistent with `slow_colour`.

Reply:
Good idea. Changed as suggested, and changed in the comment for the ball velocity too.